core/controllers/collection_editor.py

```diff
@@ -75,6 +75,8 @@ def get(self, collection_id):
 class EditableCollectionDataHandler(CollectionEditorHandler):
     """A data handler for collections which supports writing."""
 
+    GET_HANDLER_ERROR_RETURN_TYPE = 'json'
```

Comment:
Here and elsewhere, use named constants. Always use named constants, otherwise it's hard to maintain and you're opening yourself up to typos.